employees/templates/employees/common/report_list_navigation_bar.html

```diff
@@ -0,0 +1,80 @@
+{% load rest_framework %}
```

Comment:
Remember to delete such things like this after rebase on genetic views which are almost in master.

Reply:
Done